moabb/datasets/utils.py

```diff
@@ -14,14 +14,14 @@
 
 
 def dataset_search(paradigm, multi_session=False, events=None,
-                   has_all_events=False, total_classes=100, min_subjects=1,
+                   has_all_events=False, total_classes=None, min_subjects=1,
```

Comment:
here I re-implemented this so that total_classes corresponds to the exact number of classes you want or None if you don't care, so that we could make a datasets property for the ImageryNClasses paradigm which we currently can't do